Gems/Atom/RPI/Code/Source/RPI.Builders/Model/ModelAssetBuilderComponent.cpp

```diff
@@ -719,17 +742,32 @@ namespace AZ
 
                     if (sourceMesh.m_meshTangents)
                     {
+                        if (maxOldIndex > sourceMesh.m_meshTangents->GetCount())
```

Comment:
Shouldn't this (and similar below) be ">=" rather than ">"?

Reply:
Yes, that is correct. Will fix.